go/cmd/vtbackup/vtbackup.go

```diff
@@ -191,6 +208,15 @@ func main() {
 		log.Errorf("Couldn't prune old backups: %v", err)
 		exit.Return(1)
 	}
+
+	if keepAliveTimeout > 0 {
```

Comment:
New. Added for local testing, but think it could be useful in K8s context to keep process alive long enough for a Prometheus scrape interval.